x/programs/rust/examples/single_nft/scripts/copy_wasm.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if ! [[ "$0" =~ scripts/copy_wasm.sh ]]; then
+  echo "must be run from crate root"
+  exit 255
+fi
+
+root="$(pwd)"
+
+# Build the program
+build_script_path="${root}"/../../scripts/build.sh
+sh "${build_script_path}" out
+
+# Copy wasm file over
+cp out/single_nft.wasm ../../../examples/testdata/single_nft.wasm
+
+# Delete build artifacts
+rm -rf "${root}"/out
+
+echo 'Successfully copied single_nft.wasm to examples/testdata'
```

Comment:
compiled wasm should probably live in a directory in the crate vs in the golang examples dir.